ui/src/app/pages/data/concept/concept-homepage.tsx

```diff
@@ -595,9 +586,7 @@ export const ConceptHomepage = fp.flow(
                           browsePhysicalMeasurements={() =>
                             this.browseDomain(Domain.PHYSICALMEASUREMENTCSS)
                           }
-                          updating={domainsLoading.includes(
-                            Domain.PHYSICALMEASUREMENTCSS
-                          )}
+                          updating={loadingConceptCounts}
```

Comment:
Same as above, can remove `updating` and spinner from the `PhysicalMeasurementsCard` component